src/magicbox/SuggestionsManager.ts

```diff
@@ -16,63 +14,35 @@ export interface Suggestion {
   onSelect?: () => void;
 }
 
-export interface ISearchResultPreview {
-  element: HTMLElement;
-  onSelect: () => void;
-}
-
 export interface SuggestionsManagerOptions {
   suggestionClass?: string;
   selectedClass?: string;
   timeout?: number;
   previewHeaderText?: string;
 }
 
-enum Direction {
+export enum Direction {
```

Comment:
What would be the better option here? Making this exported so `ResultPreviewsManager` can use it, or making a new enum in `ResultPreviewsManager`?